app.py

```diff
@@ -0,0 +1,203 @@
+"""
+Welcome to Amity Rooms allocation System.
+Usage:
+
+    rooms_app add_person <first_name> <last_name> <role> [--wants_accommodation=N]
+    rooms_app create_room <room_name>...
+    rooms_app reallocate_person <person_identifier> <room_name> <room_type>
+    rooms_app save_state [--db=sqlite_database]
+    rooms_app load_state <database>
+    rooms_app print_allocations [--o=filename]
+    rooms_app print_allocated [--o=filename]
+    rooms_app load_people <file_path>
+    rooms_app print_room <room_name>
+    rooms_app upload_database
+    reallocate_person <first_name> <last_name> <new_room_name>
+
+
+
+Options:
+    -i, --interactive  Interactive Mode
+    -h, --help  Show this screen and exit.
+"""
+
+import sys
+import cmd
+import os
+sys.path.append('./app_classes')
+from amity import Amity
+from docopt import docopt, DocoptExit
+from termcolor import cprint, colored
+amity = Amity()
+
```

Comment:
Consider separating in built packages imports with installed packages and then local files, i.e:

```
import sys

from termcolor import cprint, colored

from amity import Amity

```

Notice the one line of separation amongst the different level imports.